docs/examples/soroban/invoke_contract_functions.md

```diff
@@ -28,12 +28,15 @@ alias Stellar.TxBuild.{
 alias Stellar.Horizon.Accounts
 alias Stellar.KeyPair
 
-contract_address =
-  "CD3HNKU3ERTEYLBBBVTSOYE4ZL2ZWV7NHLQIZRRKC4CBNMZXC7ISBXHV"
-  |> SCAddress.new()
-  |> (&SCVal.new(address: &1)).()
-function_name = SCVal.new(symbol: "hello")
-args = SCVec.new([contract_address, function_name, SCVal.new(string: "dev")])
+contract_address = SCAddress.new("CAMGSYINVVL6WP3Q5WPNL7FS4GZP37TWV7MKIRQF5QMYLK3N2SW4P3RC")
+
+function_name = "hello"
+args =
+  InvokeContractArgs.new(
+    contract_address: contract_address,
+    function_name: function_name,
+    args: SCVec.new([SCVal.new(string: "dev")])
```

Comment:
IMO, we could allow to enter a list of `SCVal` instead. Since `args` must always be a `SCVec` we can avoid that users need to create the struct.